agenda/presentation/src/main/java/com/gkp/agenda/presentation/util/Util.kt

```diff
@@ -0,0 +1,46 @@
+package com.gkp.agenda.presentation.util
+
+import java.time.LocalDate
+private const val CURRENT_AND_NEXT_SIX_DATES = 7
+
+fun getFullNameInitials(fullName: String): String {
+    val names = fullName.split(" ")
+    val initialsBuilder = StringBuilder()
+    if (names.size == 1) {
+        initialsBuilder.append(names[0].take(2).uppercase())
+        return initialsBuilder.toString()
+    }
```

Comment:
Might want to check the length of the first name as well. If it's <= 1, this will break